### DISCUSSION TITLE
Upcoming Releases and Community Interaction

### DISCUSSION THREAD

**Discussion by kschlesselmann** [awspring/spring-cloud-aws#22] [Q&A]
Hi!

I tried reaching out in gitter but it seems that no one looks at https://gitter.im/spring-cloud/spring-cloud-aws any more.

Where do you plan to communicate planned release dates? 
We'd like to upgrade to SC 2020.0.0 but it seems there's no compatible SC AWS release yet.

**Comment by MatejNedic** [comment]
Hey @kschlesselmann,
Sorry for late response. 2.3.0 will be compatible with SC 2020.0.0. Currently there are some minor changes still to be done and hopefully they will be done soon.

**Reply by kschlesselmann** [reply]
Great, thanks! Maybe just keep the expected release dates in https://github.com/awspring/spring-cloud-aws/milestones up to date. As far as I see there has been some cleanup already done ;-)

**Reply by MatejNedic** [reply]
We are still discussing and checking new ways to communicate release dates apart from milestones. Milestones will be updated and maintained in future for sure! :)

**Reply by vgropp** [reply]
Hi,
so what are your plans to communicate when to finally release a spring boot 2.4 and cloud 2020.0.0? I cannot even find open issues whats still to be done, only a new feature thats not quiet ready. Was this the intention to leave the release train to release when its ready?

**Reply by MatejNedic** [reply]
Hey @vgropp, 
we plan to release 2.3.0 soon. It will be compatible with spring cloud 2020.0.0 and Spring Boot 2.4. We have moved Github repository that's why you can't see any issues since they are still on the [old repository](https://github.com/spring-cloud/spring-cloud-aws/issues). We plan to move relevant issues from that repository to here. 
Currently for 2.3 you can check [this](https://github.com/spring-cloud/spring-cloud-aws/issues/761)
Here is [milestone for 2.3 in old repo](https://github.com/spring-cloud/spring-cloud-aws/milestone/30)

**Reply by MatejNedic** [reply]
Hey @kschlesselmann @vgropp ,
just to let you know Spring Cloud AWS 2.3.0-RC2 got released.
More can be read on https://maciejwalkowiak.com/blog/spring-cloud-aws-2-3-rc2-released/

**Reply by kschlesselmann** [reply]
@MatejNedic Nice, awesome work!

**Reply by kschlesselmann** [reply]
> If the feedback won't be overwhelming, we are aiming to release a GA on the 18th of February.

@MatejNedic Any updates on this? :-)

**Reply by MatejNedic** [reply]
Hey @kschlesselmann, we have few blockers before release: #81  and #57 . Hopefully, once they are solved we will release 2.3 if nothing else comes out.

**Comment by arunkpatra** [comment]
Does a monthly eMeet for all contributors, enthusiasts, industry partners make sense? Any medium should be good....
